parquet-column/src/main/java/org/apache/parquet/column/values/bytestreamsplit/ByteStreamSplitValuesWriter.java

```diff
@@ -102,47 +109,33 @@ public long getAllocatedSize() {
 
   public static class FloatByteStreamSplitValuesWriter extends ByteStreamSplitValuesWriter {
 
-    private static final int FLOAT_SIZE_IN_BYTES = 4;
-
     public FloatByteStreamSplitValuesWriter(int initialCapacity, int pageSize, ByteBufferAllocator allocator) {
-      super(FLOAT_SIZE_IN_BYTES, initialCapacity, pageSize, allocator);
+      super(Float.BYTES, initialCapacity, pageSize, allocator);
     }
 
     @Override
     public void writeFloat(float v) {
-      int vAsInt = Float.floatToIntBits(v);
-      byte[] bytes = new byte[FLOAT_SIZE_IN_BYTES];
-      for (int i = 0; i < FLOAT_SIZE_IN_BYTES; ++i) {
-        byte b = (byte) ((vAsInt >> (i * 8)) & 0xFF);
-        bytes[i] = b;
-      }
-      super.scatterBytes(bytes);
+      super.scatterBytes(ByteBuffer.allocate(Float.BYTES).order(ByteOrder.LITTLE_ENDIAN).putFloat(v).array());
```

Comment:
I would prefer the previous implementation because of performance reasons. You may check [ByteUtils](https://github.com/apache/parquet-mr/blob/master/parquet-common/src/main/java/org/apache/parquet/bytes/BytesUtils.java) to reuse code.